### PR TITLE
refactor: replace ternary operator with `if-else` statement

### PR DESCRIPTION
**Pull request by wsuwt**
## Description

Replace ternary operator with `if-else` statement

## Type of change

Please delete options that are not relevant.

- [x] Refactor (improves code without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/slider/index.ts [hunk 1]
```diff
@@ -1307,11 +1307,25 @@ export class Slider extends ControlElement {
...
-    const valueMax = this.range ? name === SliderDataName.from ? this.toNumber - this.minRangeNumber : this.max : this.max;
+    let valueNow = this.value;
+    let valueMin = this.min;
+    let valueMax = this.max;
 
-    const thumbStyle = { left: `${thumbPosition}%`, zIndex: this.range ? isChanged ? '4' : '3' : null };
+    if (this.range) {
+      if (name === SliderDataName.from) {
+        valueNow = this.from;
+        valueMin = this.min;
```
Comment (Sarin-Udompanish) [inline packages/elements/src/slider/index.ts]: Could we remove this line ?

File: packages/elements/src/slider/index.ts [hunk 2]
```diff
@@ -1307,11 +1307,25 @@ export class Slider extends ControlElement {
...
+        valueMax = String(this.toNumber - this.minRangeNumber);
+      }
+      else {
+        valueNow = this.to;
+        valueMin = String(this.fromNumber + this.minRangeNumber);
+        valueMax = this.max;
```
Comment (Sarin-Udompanish) [inline packages/elements/src/slider/index.ts]: Same as above

File: packages/elements/src/slider/index.ts (hunk 1 above)
Comment (wsuwt) [reply packages/elements/src/slider/index.ts]: Done.

File: packages/elements/src/slider/index.ts (hunk 2 above)
Comment (wsuwt) [reply packages/elements/src/slider/index.ts]: Done.

**Review by Sarin-Udompanish — APPROVED** [review]

**Pull request merged**